### ISSUE TITLE
Liquids

### ISSUE THREAD

**Issue by eschan145** [viblo/pymunk#211]
It would be pretty cool if we could add liquids that flow around and react to gravity. These could be common in platformer games like lava or just water.

**Comment by viblo** [comment]
Agreed! Tricky part is implementation.. What you can do now is to fake it by using lots of small circles together with some fluid like rendering on top. When I tried something like this a long time ago I wrote a metaballs shader in pyglet with lots of little circles. Worked quite well for what I think you are after, the biggest bottleneck is performance I think.

**Comment by eschan145** [comment]
They would take density for parameters so you could know how much they flow around objects.
